packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -747,21 +741,22 @@ export const selectMinAllowedInputWithFee = (state: CoinjoinRootState, accountKe
     return minAllowedInput + txSize * status.feeRateMedian;
 };
 
-export const selectIsNothingToAnonymizeByAccountKey = memoizeWithArgs(
-    (state: CoinjoinRootState, accountKey: AccountKey) => {
-        const minAllowedInputWithFee = selectMinAllowedInputWithFee(state, accountKey);
-        const account = selectAccountByKey(state, accountKey);
-        const targetAnonymity = selectTargetAnonymityByAccountKey(state, accountKey) ?? 1;
+export const selectIsNothingToAnonymizeByAccountKey = (
+    state: CoinjoinRootState,
+    accountKey: AccountKey,
+) => {
+    const minAllowedInputWithFee = selectMinAllowedInputWithFee(state, accountKey);
+    const account = selectAccountByKey(state, accountKey);
+    const targetAnonymity = selectTargetAnonymityByAccountKey(state, accountKey) ?? 1;
 
-        const anonymitySet = account?.addresses?.anonymitySet || {};
-        const utxos = account?.utxo || [];
+    const anonymitySet = account?.addresses?.anonymitySet || {};
+    const utxos = account?.utxo || [];
 
-        // Return true if all non-private funds are too small.
-        return utxos
-            .filter(utxo => (anonymitySet[utxo.address] ?? 1) < targetAnonymity)
-            .every(utxo => new BigNumber(utxo.amount).lt(minAllowedInputWithFee));
-    },
-);
+    // Return true if all non-private funds are too small.
+    return utxos
```

Comment:
This will always return new object, be aware of it

Reply:
it returns boolean 😉